### ISSUE TITLE
Tony not cutting square

### ISSUE THREAD

**Issue by magman2112** [DoESLiverpool/somebody-should#1620]
I was cutting some larger items on Tony today and noticed when I cut two parts that were mirrored that they didn't line up. Further tests shows that the vertical access is not square.

I would estimate that this axis is out by at least 5mm front to back, which just happens to be one tooth on the toothed belt drive for this axis.

I will need some help to access the belt adjustment, so I wasn't able to complete this repair today.

@DoESLiverpool/laser-maintenance

**Comment by Sean-anotherone** [comment]
Wow!  the two pulley are linked by a solid shaft, so one end must have skipped a tooth.

Thursday is the best I can make to help out.

**Comment by amcewen** [comment]
This seems to be a duplicate of #1613

**Comment by magman2112** [comment]
This was actually a separate issue, where the laser was not cutting square, which I identified whilst cutting some large parts. When I aligned two parts which were mirror images of each other, the cuts where approx 10mm out of square on a 400mm side.

@Sean-anotherone and I did some work on this a while ago though, adjusting the set screws on the shaft, which brought the cutting back to square. It’s not 100% square still, but less than 0.5mm over a 500mm side is not bad.
